monkey/infection_monkey/post_breach/actions/collect_processes_list.py

```diff
@@ -54,4 +52,6 @@ def run(self):
                 }
                 continue
 
-        return self.command, (processes, success_state)
+        # No command here; used psutil
+        self.pba_data.append(PostBreachData(self.name, "", (processes, success_state)))
```

Comment:
We return `self.command` in `clear_command_history.py`, but here we return "". We should choose a default empty value (I think self.command is good) and stick to it.